windows/setup.yml

```diff
@@ -3,8 +3,21 @@ user_message: |
   ''
 
 controller_components:
+  - organizations
+  - users
   - projects
   - job_templates
+  - roles
+
+controller_organizations:
+- name: "Helpdesk Org"
+  description: This is the Helpdesk
+
+controller_user_accounts:
+  - user: level1
+    is_superuser: false
+    password: Ansible!23
```

Comment:
I'd like to see if we can make this dynamic. Perhaps using the password from the controller credential to keep things consistent.